Rethinking_2/Chp_02.ipynb

```diff
@@ -9,7 +9,7 @@
     "import arviz as az\n",
     "import matplotlib.pyplot as plt\n",
     "import numpy as np\n",
-    "import pymc3 as pm\n",
+    "import pymc as pm\n",
```

Comment:
Issue with this chapter using pymc too: https://github.com/pymc-devs/pymc/issues/5443